app/views/mortgage_calculator/affordabilities/_form_step1.html.erb

```diff
@@ -81,21 +111,17 @@
             </span>
           <% end %>
         </div>
-        <div class="affcalc__col--tooltip">
-          <%= field_tooltip text: t("affordability.tooltips.mortgage_calculator/person.extra_income_0_html"),
-              html: {
-                id: 'tip_extra_income_0'
-              }
-          %>
-        </div>
       </div>
     <% else %>
 
       <div class="affcalc__row">
         <div class="affcalc__col">
+          <span class="affcalc__label"><%= I18n.t("affordability.titles.second_applicant") %></span>
+          <span class="visually-hidden">(displays new content below)</span>
```

Comment:
(displays new content below) <= Shouldnt this be in the YAML and also in welsh?

Reply:
This part of the view was that way originally. I can request welsh for this and add it to the YAML if you think its necessary.